lib/json_schemer/schema/base.rb

```diff
@@ -29,29 +30,40 @@ def merge(
         :eol => '\z'
       }.freeze
 
+      INSERT_DEFAULT_PROPERTY = -> (data, property, property_schema) {
```

Comment:
Same here. Let's make this a proc to match the ref resolver constants. I usually prefer `do ... end` for multiline blocks as well.